sec2_the_ruby_programming_language/2.5_methods.rb

```diff
@@ -0,0 +1,32 @@
+def multiply(first, second)
```

Comment:
Can keep things consistent?
Here is:
```
(first, second)
```
bottom is
```
(first,second)
```

Choose one and stick with it in the entire codebase